wikichange/src/content/content.js

```diff
@@ -106,6 +106,32 @@ const highlightPersistentContent = (text, color) => {
     }
 };
 
+/* Highlights the words that are given with context. No support for links yet */
+const highlightContentWithContext = (json, color) => {
+    let foundIndex = -1;
+    let controlIndex = 0;
+    let highlight = json["highlight"];
+    let before = json["content_before"];
+    let after = json["content_after"];
+    while ((foundIndex = innerHTML.indexOf(highlight, controlIndex)) != -1) {
+        let afterText = innerHTML.substring(foundIndex + highlight.length, foundIndex + highlight.length + after.length);
+        let beforeText = innerHTML.substring(foundIndex - before.length, foundIndex);
+        console.log("After " + afterText);
+        console.log("Before " + beforeText);
```

Comment:
did u intend to keep these logs?

Reply:
For now it can help us figure out why it didn't highlight some stuff if they are html